src/machine.rs

```diff
@@ -178,6 +178,20 @@ impl OlmMachine {
         future_to_promise(async move { Ok(me.display_name().await?) })
     }
 
+    /// Mark all tracked users as dirty.
+    ///
+    /// All users *whose device lists we are tracking* are flagged as needing a
+    /// key query. Users whose devices we are not tracking are ignored.
+    #[wasm_bindgen(js_name = "markAllTrackedUsersAsDirty")]
+    pub fn mark_all_tracked_users_as_dirty(&self) -> Promise {
```

Comment:
did you give any particular thought to where in the file to add this method? It looks a bit random at first glance, but I've not taken the time to figure out if there is anywhere better.

Reply:
somewhere near the other functions dealing with tracked users seems sensible?